src/adapters/MongoStorageAdapter.js

```diff
@@ -113,9 +132,47 @@ const MongoStorageAdapter = function (uuid) {
 			cb(err, { status: "failed" });
 		});
 	};
-};
 
+	/**
+	 * Get the prefix used to filter keys for particular topics and key prefixes.
+	 *
+	 * @param {object} params
+	 * @param {string} params.topic The topic
```

Comment:
```suggestion
	 * @param {string} params.topic The topic (optional).
```
To go with implementation suggestion below